build/localnet/Tiltfile

```diff
@@ -314,6 +314,16 @@ for x in range(localnet_config["full_nodes"]["count"]):
 
     k8s_resource("full-node-%s-pocket" % formatted_number, labels=["pocket-full-nodes"])
 
+# Provisions anvil node and service for E2E testing
+k8s_yaml(
+    helm(
```

Comment:
I'm getting an error coming form this line when trying to run this code locally . 

```
Running: helm template anvil-node /Users/shakezula/dev/pokt/pocket/charts/anvil --include-crds
Error: Traceback (most recent call last):
  /Users/shakezula/dev/pokt/pocket/build/localnet/Tiltfile:319:9: in <toplevel>
Error in helm: command "helm template anvil-node /Users/shakezula/dev/pokt/pocket/charts/anvil --include-crds" failed.
error: exit status 1
stdout: ""
stderr: "Error: template: anvil/templates/deployment.yaml:8:20: executing \"anvil/templates/deployment.yaml\" at <.Values.autoscaling.enabled>: nil pointer evaluating interface {}.enabled\n\nUse --debug flag to render out invalid YAML\n"
make: *** [localnet_down] Error 1
```
`make localnet_up` and `make localnet_down` are both reporting this error. Have you seen this error before, by chance?